<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGAL - VSA demo</string>
  </property>
  <property name="windowIcon">
   <iconset resource="VSA_demo.qrc">
    <normaloff>:/cgal/icons/resources/cgal_logo.xpm</normaloff>:/cgal/icons/resources/cgal_logo.xpm</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="Viewer" name="viewer" native="true">
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionLoadPolyhedron"/>
    <addaction name="actionSaveApproximation"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionViewPolyhedron"/>
    <addaction name="actionViewWireframe"/>
    <addaction name="actionViewBoundary"/>
    <addaction name="separator"/>
    <addaction name="actionViewProxies"/>
    <addaction name="actionViewAnchors"/>
    <addaction name="actionViewApproximation"/>
   </widget>
   <widget class="QMenu" name="menuAlgorithms">
    <property name="title">
     <string>&amp;Metric</string>
    </property>
    <addaction name="actionL21"/>
    <addaction name="actionL2"/>
    <addaction name="actionCompact"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionCopySnapshot"/>
    <addaction name="actionSaveSnapshot"/>
   </widget>
   <widget class="QMenu" name="menuOperation">
    <property name="title">
     <string>&amp;Operations</string>
    </property>
    <addaction name="actionApproximation"/>
    <addaction name="separator"/>
    <addaction name="actionSeeding"/>
    <addaction name="actionFit"/>
    <addaction name="actionMeshing"/>
    <addaction name="separator"/>
    <addaction name="actionAdd"/>
    <addaction name="actionTeleport"/>
    <addaction name="actionSplit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAlgorithms"/>
   <addaction name="menuOperation"/>
   <addaction name="menuView"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionLoadPolyhedron">
   <property name="text">
    <string>&amp;Load polyhedron...</string>
   </property>
  </action>
  <action name="actionViewWireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Wireframe</string>
   </property>
   <property name="toolTip">
    <string>View wireframe</string>
   </property>
   <property name="shortcut">
    <string>Alt+W</string>
   </property>
  </action>
  <action name="actionSaveSnapshot">
   <property name="text">
    <string>&amp;Save snapshot</string>
   </property>
  </action>
  <action name="actionCopySnapshot">
   <property name="text">
    <string>&amp;Copy snapshot</string>
   </property>
  </action>
  <action name="actionViewBoundary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Boundary</string>
   </property>
   <property name="toolTip">
    <string>View proxy boundary</string>
   </property>
   <property name="shortcut">
    <string>Alt+B</string>
   </property>
  </action>
  <action name="actionViewAnchors">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Anchors</string>
   </property>
   <property name="toolTip">
    <string>View anchors</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionViewApproximation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Appro&amp;ximation</string>
   </property>
   <property name="toolTip">
    <string>View approximation</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionViewPolyhedron">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Polyhedron</string>
   </property>
   <property name="toolTip">
    <string>View polyhedron</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionSaveApproximation">
   <property name="text">
    <string>&amp;Save approximation</string>
   </property>
   <property name="toolTip">
    <string>Save approximation</string>
   </property>
  </action>
  <action name="actionSeeding">
   <property name="text">
    <string>&amp;Seeding</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionFit">
   <property name="text">
    <string>&amp;Fit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionTeleport">
   <property name="text">
    <string>&amp;Teleport</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionMeshing">
   <property name="text">
    <string>&amp;Meshing</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionL21">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>L2&amp;1</string>
   </property>
  </action>
  <action name="actionL2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>L&amp;2</string>
   </property>
  </action>
  <action name="actionCompact">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Compact</string>
   </property>
  </action>
  <action name="actionApproximation">
   <property name="text">
    <string>Appro&amp;ximation</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionViewProxies">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;L21 Proxies</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionSplit">
   <property name="text">
    <string>Split</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Viewer</class>
   <extends>QWidget</extends>
   <header>Viewer.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="VSA_demo.qrc"/>
 </resources>
 <connections/>
</ui>
